docs/tuning_fledge.rst

```diff
@@ -789,11 +789,11 @@ The log purging is perhaps the simpler of the two purge process to discuss as it
 | |PurgeSystemConfig| |
 +---------------------+
 
-The configuration options merely allow you to set the number of days worth of data that should be retained for each of the three log categories. The important consideration here is that the various logs should not be allowed to grow to such an extent that you risk exhausting the storage system, but that should should retain sufficient information to be able to examine enough history of the system.
+The configuration options merely allow you to set the number of days worth of data that should be retained for each of the three log categories; audit, tasks and statistics. The important consideration here is that the various logs should not be allowed to grow to such an extent that you risk exhausting the storage system, but that should retain sufficient information to be able to examine enough history of the system.
```

Comment:
- three log categories; audit, tasks and statistics -> three log categories: audit, tasks and statistics (i.e. colon)
- risk exhausting the storage system, but that should retain -> risk exhausting the storage system, but should retain